packages/devtools_app/lib/src/ui/service_extension_widgets.dart

```diff
@@ -136,13 +136,10 @@ class _ServiceExtensionButtonGroupState
           : description.disabledTooltip,
       waitDuration: tooltipWait,
       preferBelow: false,
-      child: Padding(
-        padding: const EdgeInsets.symmetric(horizontal: defaultSpacing),
```

Comment:
you may still need the padding when the text is shown. If you resize the window to be wide does this look right?

Reply:
Ah right, adding back